src/components/tasks/RegisterTask.vue

```diff
@@ -49,7 +49,7 @@
             </section>
             <!--Task URL-->
             <section class="form-section">
-              <div>
+              <div class="taskURLs">
```

Comment:
I'd like to use kebab case for css classes.

```suggestion
              <div class="task-urls">
```